examples/less-extract/package.json

```diff
@@ -2,13 +2,20 @@
   "name": "less-extract",
   "private": true,
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack",
+    "build": "npm run build:dev && npm run build:pro",
```

Comment:
I think we should just build once for prod

Reply:
This is an example, and the LESS (and SCSS) transpiler works diferently in dev and prod.